Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -97,8 +97,29 @@ def get_config ():
       'ecptr'     :   'EC/ecregionpointer.bin',
       'gbe'       :   'GBE/gbe.bin',
       'bpm'       :   'bpmgen2',
+      'bpm_params':   'bpmgen2/Example.bpDef',
     }
     return cfg_var
+
+def get_bpmgen2_params_change_list ():
+    params_change_list = []
+    params_change_list.append ([
+      # variable                | value |
+      # ===================================
+      ('PlatformRules',         'CFL Client'),
+      ('BpmStrutVersion',       '0x10'),
+      ('BpmRevision',           '0x01'),
+      ('BpmRevocation',         '1'),
+      ('AcmRevocation',         '2'),
+      ('NEMPages',              '3'),
+      ('IbbFlags',              '0x2'),
+      ('TxtInclude',            'FALSE'),
+      ('BpmSigScheme',          '0x14:RSASSA'),
+      ('BpmSigPubKey',          'keys\pubkey.pem'),
+      ('BpmSigPrivKey',         'keys\privkey.pem'),
+      ])
+    return params_change_list
+
```

Comment:
Not sure if we could a generic StitchIfwi.py by moving this platform specific config to a config file and let StitchIfwi.py 
 consume it. any idea?